extendr-api/src/robj/into_robj/repeat_into_robj.rs

```diff
@@ -86,3 +86,49 @@ impl_synonym_type!(Rfloat, f64);
 impl_synonym_type!(&Rfloat, f64);
 impl_synonym_type!(Rint, i32);
 impl_synonym_type!(&Rint, i32);
+
+// since `Rstr` does not implement `Scalar` (due to lifetime conflicts),
+// it requires manual implementation
+
+impl ToVectorValue for Rstr {
+    fn sexptype() -> SEXPTYPE {
+        SEXPTYPE::STRSXP
+    }
+
+    fn to_sexp(&self) -> SEXP
+    where
+        Self: Sized,
+    {
+        unsafe { self.robj.get() }
+    }
+}
+
+impl ToVectorValue for &Rstr {
+    fn sexptype() -> SEXPTYPE {
+        SEXPTYPE::STRSXP
+    }
+
+    fn to_sexp(&self) -> SEXP
+    where
+        Self: Sized,
+    {
+        unsafe { self.robj.get() }
+    }
+}
+
+impl ToVectorValue for Option<Rstr> {
+    fn sexptype() -> SEXPTYPE {
+        SEXPTYPE::STRSXP
+    }
+
+    fn to_sexp(&self) -> SEXP
+    where
+        Self: Sized,
+    {
+        if let Some(s) = self {
+            unsafe { s.robj.get() }
+        } else {
+            unsafe { R_NaString }
+        }
+    }
+}
```

Comment:
I think this alone can be merged. The rest of the PR is great, but would need a lot of discussion I think. Should we port this over to a smaller PR ?

Reply:
I don't think this is correct, unfortunately. It looks useful, but I think it is flawed. We need to brainstorm about this more before I can fully commit to an explaination or a strategy on this.

Basically, the `ToVectorValue` stuff assumes certain things about the extendr API, and these assumptions are not complete.. Sorry, I need to write a more thorough explaination of the issues here.. and somewhere more central.